app/default_overrides_test.go

```diff
@@ -26,7 +26,7 @@ func Test_newGovModule(t *testing.T) {
 
 	want := []types.Coin{{
 		Denom:  BondDenom,
-		Amount: types.NewInt(1000000000),
+		Amount: types.NewInt(10000000000),
```

Comment:
[nit] for readability b/c it's easy to mess these values up without separators.

```suggestion
		Amount: types.NewInt(10_000_000_000),
```